src/components/FlagSelect.tsx

```diff
@@ -1,3 +1,89 @@
-export function Welcome() {
-    return <h1>something</h1>;
-  }
\ No newline at end of file
+import React, { useEffect, useState } from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import List from "@material-ui/core/List";
+import ListItem from "@material-ui/core/ListItem";
+import DialogTitle from "@material-ui/core/DialogTitle";
+import Dialog from "@material-ui/core/Dialog";
+import Australia from "/home/ishfaq/training-app/src/flags/Australia.png"
+import america from "/home/ishfaq/training-app/src/flags/america.png"
+import { Typography } from "@material-ui/core";
```

Comment:
altogether /core imports like this
import { Typography, List,  ListItem, DialogTitle, Dialog } from "@material-ui/core";